features/client-hints/main.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>Client Hints</title>
+</head>
+
+<body>
+  <p><a href="../index.html">[Home]</a></p>
+
+  <p>This site asks browser for all client hints possible via headers and via JS.</P>
+
+  <h3>Hints requested via `accept-ch`:</h3>
+  <table id="headers"></table>
+
+  <h3>Hints requested via `navigator.userAgentData`:</h3>
+  <table id="js"></table>
+
+  <script src="./all-ch.js"></script>
+  <script>
+    /* global ALL_CH, ALL_CH_JS */
+
+    const headerTable = document.getElementById('headers');
+
+    function createRow (name, value, extraHTML) {
+        const tr = document.createElement('tr');
+        const tdName = document.createElement('td');
+        const tdValue = document.createElement('td');
+
+        let valueHTML = null;
+
+        if (typeof value === 'undefined') {
+            valueHTML = '❌ (<i>missing</i>)';
+        } else if (value.length === 0) {
+            valueHTML = '❌ (<i>empty</i>)';
+        }
+
+        tdName.innerText = name;
+        tdValue.innerText = value;
+        if (valueHTML) {
+            tdValue.innerHTML = valueHTML;
+        }
+        if (extraHTML) {
+            const span = document.createElement('span');
+            span.innerHTML = extraHTML;
+            tdValue.appendChild(span);
+        }
+
+        tr.appendChild(tdName);
+        tr.appendChild(tdValue);
+
+        return tr;
+    }
+
+    fetch('/reflect-headers')
+        .then(r => r.json()).then(data => {
+            ALL_CH.forEach(hint => {
+                const value = data.headers && data.headers[hint];
+
+                headerTable.appendChild(createRow(hint, value));
+            });
+
+            Object.keys(data.headers).forEach(header => {
```

Comment:
Besides checking for hints that we requested, we are also looking for any hints that we haven't requested and flagging them.